components/About/index.jsx

```diff
@@ -1,25 +1,29 @@
-import StyleGuide from '../Styleguide'
 import clsx from 'clsx'
+import PropTypes from 'prop-types'
 import styles from './styles.module.scss'
 
-const About = () => {
+const About = (props) => {
```

Comment:
Are ESLint errors showing up for you? 
![image](https://user-images.githubusercontent.com/15239884/115269573-509d1880-a0f0-11eb-8a09-ed45cea96e0a.png)

Try destructuring for props: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#object_destructuring.

In this case, `props` is an object that looks like:
```
{
  heading: 'About',
  image: 'images/guava-9.png',
  ...
}
```

`console.log(props)` if you want to confirm in the browser (but don't commit)

Since it's an object, you're be able to destructure/"pull out" the properties like so:
`const { heading, image, *other props here* } = props`

So now can use `heading`, `image`, etc. as regular constants instead of having to use `props.whatever`

Once you have props destructuring set up in the component body, you can move it directly into the params:

```
const About = (props) => {
  const { heading, image, *other stuff* } = props
```
becomes
```
const About = ({ heading, image, *other stuff* } ) => {
  // no more use of 'props'
```